### DISCUSSION TITLE
Any way to propagate elevation ?

### DISCUSSION THREAD

**Discussion by Grensorcer** [Flow-Launcher/Flow.Launcher#2978] [Q&A] [answered]
Hi !
I have been looking at multiple launchers on Windows and have not found any that starts my program with the same elevation level as I started the launcher with. Example: Flow requires ctrl + shift + click to start a program as administrator.

I can't find much information on why that's usually the case, would that be a possibility in Flow at some point ?

**Comment by jjw24** [comment]
>have not found any that starts my program with the same elevation level as I started the launcher with. Example: Flow requires ctrl + shift + click to start a program as administrator.

With flow, programs are executed as the same user you started flow with and likewise if you started with admin account, so should be the same level.

**Reply by Grensorcer** [reply]
Indeed it seems I was thorough enough when testing it out.
The first application I made a test with was "Windows Terminal", which does not get elevated rights for some reason.
I am still able to get elevated right when starting other applications.

Any idea why some applications would not get started as administrator when others do ?

**Reply by Grensorcer** [reply, marked as answer]
Okay I've found out that Microsoft Store applications can't be run as admin from an elevated Flow program. Installing Windows Terminal manually instead fixed my problem !
